docs/core/diagnostics/collect-dumps-crash.md

```diff
@@ -12,13 +12,13 @@ The following table shows the environment variables you can configure for collec
 
 | Environment variable | Description | Default value |
 |----------------------|-------------|---------------|
-|`COMPlus_DbgEnableMiniDump` or `DOTNET_DbgEnableMiniDump`|If set to 1, enable core dump generation.|0|
-|`COMPlus_DbgMiniDumpType` or `DOTNET_DbgMiniDumpType`|Type of dump to be collected. For more information, see [Types of mini dumps](#types-of-mini-dumps).|2 (`Heap`)|
-|`COMPlus_DbgMiniDumpName` or `DOTNET_DbgMiniDumpName`|Path to a file to write the dump to. Ensure that the user under which the dotnet process is running has write permissions to the specified directory.|`/tmp/coredump.<pid>`|
-|`COMPlus_CreateDumpDiagnostics` or `DOTNET_CreateDumpDiagnostics`|If set to 1, enables diagnostic logging of dump process.|0|
-|`COMPlus_EnableCrashReport` or `DOTNET_EnableCrashReport`|(Requires .NET 6 or later; not supported on Windows.)<br/>If set to 1, the runtime generates a JSON-formatted crash report that includes information about the threads and stack frames of the crashing application. The crash report name is the dump path or name with *.crashreport.json* appended.| |
-|`COMPlus_CreateDumpVerboseDiagnostics` or `DOTNET_CreateDumpVerboseDiagnostics`|(Requires .NET 7 or later.)<br/>If set to 1, enables verbose diagnostic logging of the dump process.|0|
-|`COMPlus_CreateDumpLogToFile` or `DOTNET_CreateDumpLogToFile`|(Requires .NET 7 or later.)<br/>The path of the file to which the diagnostic messages should be written. | If unset, the diagnostic messages are written to the console of the crashing application. |
+|`DOTNET_DbgEnableMiniDump` or `COMPlus_DbgEnableMiniDump`|If set to 1, enable core dump generation.|0|
+|`DOTNET_DbgMiniDumpType` or `COMPlus_DbgMiniDumpType`|Type of dump to be collected. For more information, see [Types of mini dumps](#types-of-mini-dumps).|2 (`Heap`)|
+|`DOTNET_DbgMiniDumpName` or `COMPlus_DbgMiniDumpName`|Path to a file to write the dump to. Ensure that the user under which the dotnet process is running has write permissions to the specified directory.|`/tmp/coredump.<pid>`|
+|`DOTNET_CreateDumpDiagnostics` or `COMPlus_CreateDumpDiagnostics`|If set to 1, enables diagnostic logging of dump process.|0|
+|`DOTNET_EnableCrashReport` or `COMPlus_EnableCrashReport`|(Requires .NET 6 or later; not supported on Windows.)<br/>If set to 1, the runtime generates a JSON-formatted crash report that includes information about the threads and stack frames of the crashing application. The crash report name is the dump path or name with *.crashreport.json* appended.| |
+|`DOTNET_CreateDumpVerboseDiagnostics` or `COMPlus_CreateDumpVerboseDiagnostics`|(Requires .NET 7 or later.)<br/>If set to 1, enables verbose diagnostic logging of the dump process.|0|
```

Comment:
I would drop "Requires .NET 7 or later.". We do not need to maintain docs that are out of support for a while.